src/core_modules/capture-core/components/Pages/MainPage/WorkingListsCommon/actions/workingListsCommon.actions.js

```diff
@@ -45,8 +45,8 @@ export const fetchTemplates =
     (programId: string, listId: string) =>
         actionCreator(workingListsCommonActionTypes.TEMPLATES_FETCH)({ programId, listId });
 
-export const fetchTemplatesSuccess = (templates: Array<any>, programId: string, listId: string) =>
-    actionCreator(workingListsCommonActionTypes.TEMPLATES_FETCH_SUCCESS)({ templates, programId, listId });
+export const fetchTemplatesSuccess = (templates: Array<any>, defaultTemplateId: string, listId: string) =>
```

Comment:
Perhaps if the types are the same as the ones in the file `MainPage/EventWorkingLists/types/eventWorkingLists.types.js` we could annotate this

Reply:
This will be used for both events and tracked entity instances and the data will have a different structure. Leaving this one for later.

Reply:
But then we can add `TypeOne | TypeTwo` for these structure. Now leaving this `any` there is a high chance that when we are adding the second type this wont throw an error therefore will be forgotten. 

I think adding the template type on this will help us follow up on adding the second type later on.

Reply:
Thinking that maybe a more generic type is the better solution here (With generics maybe). Let me think about this one and deal with it later.